routes/root.js

```diff
@@ -191,29 +191,53 @@ route.get('/claims/view', (req, res) => {
         })
       })
 
-      res.render('pages/claims/view', {
-        prevPage: options.page - 1,
-        nextPage: options.page + 1,
-        pagination: pagination,
-        isFirstPage: options.page == 1,
-        isLastPage: lastPage == options.page,
-        page: options.page,
-        size: options.size,
-        claims: data[1].rows,
-        menu: {
-          claims_view: 'active',
-          claims: 'claims-active'
-        },
-        status: options.status,
-        menuH,
-        current,
-        filter: filter,
-        filterproj: filterproj,
-        username: options.username,
-        projectname: options.projectname,
-        minbounty: options.minbounty,
-        maxbounty: options.maxbounty
+      const conflictCounts = {}
+
+      const conflictsLoader = async() => {
+        for (claim of data[1].rows) {
+          pullUrlDetail = getUrlDetails(claim["pullUrl"])
+          issueUrlDetail = getUrlDetails(claim["issueUrl"])
+          conflictKey = issueUrlDetail.project + '/' + pullUrlDetail.type + '/' +issueUrlDetail.id
+          if(conflictCounts[conflictKey] === undefined) {
+            const c = await du.getConflictsCount(claim,issueUrlDetail,pullUrlDetail.type)
```

Comment:
How can a query inside a loop be a single query. Try to write a separate raw query for this.

Reply:
ok so you mean, there is no need for a loop just a query wich groups all conflicts for all claims. ??

Reply:
Yes